src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -34,8 +37,7 @@ case class Instance(
 
   val runSpecId: PathId = instanceId.runSpecId
 
-  // An instance has to be considered as Reserved if at least one of its tasks is Reserved.
-  def isReserved: Boolean = tasksMap.values.exists(_.status.condition == Condition.Reserved)
+  lazy val isReserved: Boolean = state.condition == Condition.Reserved
```

Comment:
I think renaming to suspended actually makes sense because isReserved is kind of misleading now - because it's not obvious that it's the terminal state and not the pre-launch state, that is in a sense reserved too...

Reply:
Do you want to follow up with a renaming?

Reply:
Do you think it's worth it? The reserved state will go anyway, right? It will be state: Killed, target: Suspended

Reply:
oh I did not get that from the comment. I put the comment back as well. Thanks!

Too bad there is no test that would actually catch this regression :)